communitiesuk/funding-service-design-fund-store
# Fs 2956 nstf weightings

### Change description
Updating section weightings as per ticket and prototype.
Skills -> 15%
Proposal -> 40%

- [n/a] README and other documentation has been updated / added (if needed)


### How to test
Execute script FS2956_ns_weightings and see the weightings change on the tasklist


### Screenshots of UI changes (if applicable)
![Screenshot 2023-06-06 at 11 18 09](https://github.com/communitiesuk/funding-service-design-fund-store/assets/1729216/faf585ee-f060-4907-b4e7-b1428780c645)
